django/conf/project_template/project_name/settings.py

```diff
@@ -82,6 +82,22 @@
 }
 
 
+# Password validation
+# https://docs.djangoproject.com/en/{{ docs_version }}/ref/settings/#auth-password-validators
+
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
```

Comment:
Because it's a predefined identifier, similar to ENGINE and NAME in the [DATABASES](https://docs.djangoproject.com/en/1.8/ref/settings/#databases) setting and BACKEND and DIRS in [TEMPLATES](https://docs.djangoproject.com/en/1.8/ref/settings/#templates).